vault/resource_gcp_secret_impersonated_account_test.go

```diff
@@ -0,0 +1,191 @@
+package vault
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+	"golang.org/x/oauth2/google"
+)
+
+// This test requires that you pass credentials for a user or service account having the IAM rights
+// listed at https://www.vaultproject.io/docs/secrets/gcp/index.html for the project you are testing
+// on. The credentials must also allow setting IAM permissions on the project being tested.
+func TestGCPSecretImpersonatedAccount(t *testing.T) {
```

Comment:
If this is a Vault 1.13 feature, this test will fail until Vault 1.13 is released and we can run tests against it in CI; in order to not break CI, we can add this line at the start of the test function to skip this test as it is meant to be run on an upcoming version of Vault.

```suggestion
func TestGCPSecretImpersonatedAccount(t *testing.T) {
    testutil.SkipTestEnvSet(t, testutil.EnvVarSkipVaultNext)
```

Reply:
Neat